client/src/js/components/bhBreadcrumb.js

```diff
@@ -75,7 +75,7 @@ function BreadcrumbController() {
    * Dropdowns definition
    * @example
    * vm.bcDropdowns = [
-   *  {
+   *  {\';
```

Comment:
This should probably be removed before this is merged.  It will break our documentation generation.